README.md

```diff
@@ -42,11 +54,14 @@ ansible-lint site.yml
 
 ## Setting up a new staging server
 
-Add the new server to the `hosts` file. Then run:
+1. Add the new server to the `hosts` file.
+2. Get the root user password
+3. Optionally add public key(s) to `files/admin-ssh-keys`
+4. Then:
 
 ```
-ansible-playbook -i hosts -l staging2.ceresfairfood.org.au site.yml -u root
-ssh members.ceresfairfood.org.au@staging2.ceresfairfood.org.au -A
+ansible-playbook -i hosts -l staging2.ceresfairfood.org.au site.yml -u root -k
```

Comment:
What means `-k`? Is there a long version?

Reply:
Yep, will update to `--ask-pass`